### PR TITLE
feat(thumbnailGridBlock): minor design changes

### PR DESCRIPTION
**Pull request by celine-s**
ticket: https://app.clickup.com/t/8677h51hv

* fix flyout placement
* rename some labels
* improve RTE performance by only using updateValueOnChange on the placeholder
* hide toolbar in viewmode

### CODE REVIEW

File: packages/shared/src/components/BlockItemWrapper/BlockItemWrapper.tsx [hunk 1]
```diff
@@ -63,6 +63,7 @@ export const BlockItemWrapper = ({
                     !shouldHideWrapper &&
                         'group-hover:tw-opacity-100 group-focus:tw-opacity-100 focus-within:tw-opacity-100',
                     (isFlyoutOpen || shouldBeShown) && 'tw-opacity-100',
+                    shouldHideWrapper && 'tw-hidden',
```
Comment (ragi96) [inline packages/shared/src/components/BlockItemWrapper/BlockItemWrapper.tsx:66]: Just a question why is this needed? Or was it a bug before?

Comment (ragi96) [reply packages/shared/src/components/BlockItemWrapper/BlockItemWrapper.tsx:66]: Ok great, thanks for clarification

**Review by ragi96 — APPROVED** [review]
🚀

**Review by fulopdaniel — APPROVED** [review]
